src/runtime-rs/src/shim-proto/build.rs

```diff
@@ -4,9 +4,13 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
This would be easier review if the patches were broken on logical grounds, rather than on the actual history of development.  You've just introduced this code, and now it's being reworked.

Reply:
Fixed in the second revision: rebased the patches by logical separations